### PR TITLE
update view history order

### PR DESCRIPTION
**Pull request by namuetn**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by namuetn** [comment]
ready

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -266,7 +266,9 @@ public function createOrder(OrderRequest $request)
         if (auth()->check()) {
             $user = auth()->user();
         }
-        $orders = $user->orders;
+
+        $ordersSuccess = $user->orders->where('status', config('order.success'));
+        $ordersPending = $user->orders->where('status', config('order.pending'));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/ProductController.php]:
```suggestion
$orders = auth()->user()->orders;
}
        $ordersSuccess = $orders->where('status', config('order.success'));
        $ordersPending = $orders->where('status', config('order.pending'));
```

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -18,9 +18,10 @@ public function index()
             $user = auth()->user();
         }
 
-        $orders = $user->orders;
+        $ordersSuccess = $user->orders->where('status', config('order.success'));
+        $ordersPending = $user->orders->where('status', config('order.pending'));
```
Comment (huyenttm-0901) [inline app/Http/Controllers/UserController.php]: tương tự

**Comment by namuetn** [comment]
ready

**Pull request merged**